### DISCUSSION TITLE
REI Compability with crafting functanility through other mods.

### DISCUSSION THREAD

**Discussion by Zelioth** [shedaniel/RoughlyEnoughItems#1311] [Q & A]
I love REI but my MAJOR issue is that there is NO SUPPORT for crafting grids from other mods.

let me explain. for one I have REI installed with it's compatibility mod both on their latest version for 1.19.3. This issue was also found when trying to use REI in my 1.19.2 pack Friends United. 

Let me give an example. in JEI you open ANY crafting grid from any mod and you can open a recipe from the JEI item list and you can click a little + and if you have the items it auto puts them in the crafting grid. IF you do not have the items it will show an error message saying that you don't have what is needed for that item. IF it is not a crafting grid the + won't even show up

NOW lets talk about REI. I have TONS of storage mods and mods that add crafting table variants to the mod pack and NOT A SINGLE ONE allows that functionality but the + in REI is there.  well the ! but it says " Move items is not supported with this recipe and this continer" and it says it in EVERY crafting grid no matter what I have in my inventory  from other mod variants. 

Toms Simple Storage
AE2
Dank Stoarge. 

All the crafting grids do not work even if the items are in my inventory or in the system itself. Why is there no support for other mods storage options. 

I'll provide a screenshot for example.
![REI Toms Simple Storage Crafting Terminal](https://user-images.githubusercontent.com/77481431/210856130-cce8e085-9c1f-4cdb-b485-57067fbbeb87.png)

![JEI Examples](https://user-images.githubusercontent.com/77481431/210856116-e601c205-803f-468b-a74a-2b424a5e7da9.png)

**Comment by Zelioth** [comment]
Also on the left side where it shows multiple of the crafting grids that are present to be used in the mod pack why is REI empty. when I have atleast 4 other mods that contain crafting terminals but REI only sees three items

**Comment by Zelioth** [comment]
[rei.log](https://github.com/shedaniel/RoughlyEnoughItems/files/10356370/rei.log)
[rei-issues.log](https://github.com/shedaniel/RoughlyEnoughItems/files/10356371/rei-issues.log)
[latest.log](https://github.com/shedaniel/RoughlyEnoughItems/files/10356372/latest.log)

**Reply by shedaniel** [reply]
What are your REI, REIPC and AE2 versions? Can you try updating them all?

**Reply by Zelioth** [reply]
all the latest for what is released on 1.19.3. I update every time one is up and I check daily. 

REI V 10.0.581
REI C V 10.0.45

**Reply by Zelioth** [reply]
Any update on if this is a bug or how to enable compat for other storage mods?
